cmake/schemes/url_sha1_autogen_autotools.cmake.in

```diff
@@ -62,9 +62,12 @@ if(NOT default_pkgconfig)
   set(DEPENDS_ON_PACKAGES ${DEPENDS_ON_PKGCONFIGS})
 endif()
 
+set(PKG_GENERATE_SHARED "${BUILD_SHARED_LIBS}")
```

Comment:
We need it there too (?):
- https://github.com/ruslo/hunter/blob/v0.23.158/cmake/schemes/url_sha1_autotools.cmake.in#L67-L75
- https://github.com/ruslo/hunter/blob/v0.23.158/cmake/projects/tcl/schemes/url_sha1_tcl_autotools.cmake.in#L92-L100
- https://github.com/ruslo/hunter/blob/v0.23.158/cmake/projects/xcb/schemes/xcb.cmake.in#L72-L79
- https://github.com/ruslo/hunter/blob/v0.23.158/cmake/projects/x264/schemes/url_sha1_x264.cmake.in#L344-L349

Reply:
I just commented this somewhere else. I am in the process.